src/test/java/uk/ac/cam/cl/dtg/segue/api/managers/GroupManagerTest.java

```diff
@@ -130,7 +132,7 @@ final void groupManager_createValidGroup_aGroupShouldBeCreated() {
       assertEquals(someGroupName, capturedGroup.getValue().getGroupName());
       assertInstanceOf(Date.class, capturedGroup.getValue().getCreated());
 
-    } catch (SegueDatabaseException e) {
+    } catch (SegueDatabaseException | NoUserException e) {
       fail("No exception expected");
       e.printStackTrace();
```

Comment:
Not really your fault, but since you're changing code in this area, could you remove this line and instead just pass `e` as the second parameter of `fail`, please?